src/applications/personalization/dashboard/components/apply-for-benefits/SavedApplications.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchEDUPaymentInformation as fetchEDUPaymentInformationAction } from '@@profile/actions/paymentInformation';
+import { VA_FORM_IDS } from '~/platform/forms/constants';
+import { isVAPatient, isLOA3, selectProfile } from '~/platform/user/selectors';
+
+import { filterOutExpiredForms } from '~/applications/personalization/dashboard/helpers';
+
+import { getEnrollmentStatus as getEnrollmentStatusAction } from '~/applications/hca/utils/actions';
```

Comment:
Should we move this action into our own application folder to prevent this cross app import?